manifests/kubeadm_init.pp

```diff
@@ -25,4 +25,12 @@
     unless      => $unless_init,
   }
 
+  # This prevents a known race condition https://github.com/kubernetes/kubernetes/issues/66689
+  exec { 'wait for default serviceaccount creation':
+    command     => 'kubectl -n default get serviceaccount default -o name'
```

Comment:
might need to put a comma on the end here

Reply:
Fixed.